src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfig.tsx

```diff
@@ -12,18 +12,71 @@ import {
 } from '@chakra-ui/react'
 import type { Asset } from '@shapeshiftoss/types'
 import { bn, bnOrZero } from '@shapeshiftoss/utils'
-import { noop } from 'lodash'
-import { useCallback, useMemo, useState } from 'react'
-import { Amount } from 'components/Amount/Amount'
+import { useCallback, useMemo, useRef, useState } from 'react'
+import type { NumberFormatValues } from 'react-number-format'
+import NumberFormat from 'react-number-format'
 import { StyledAssetMenuButton } from 'components/AssetSelection/components/AssetMenuButton'
 import { SwapIcon } from 'components/Icons/SwapIcon'
-import { RawText, Text } from 'components/Text'
+import { Text } from 'components/Text'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+import { assertUnreachable } from 'lib/utils'
+import { allowedDecimalSeparators } from 'state/slices/preferencesSlice/preferencesSlice'
 
-const EXPIRY_TIME_PERIODS = ['1 hour', '1 day', '3 days', '7 days', '28 days'] as const
+import { AmountInput } from '../../TradeAmountInput'
+
+enum PriceDirection {
+  Default = 'default',
+  Reversed = 'reversed',
+}
+
+enum PresetLimit {
+  Market = 'market',
+  OnePercent = 'onePercent',
+  TwoPercent = 'twoPercent',
+  FivePercent = 'fivePercent',
+  TenPercent = 'tenPercent',
+}
+
+enum ExpiryOption {
+  OneHour = '1 hour',
+  OneDay = '1 day',
+  ThreeDays = '3 days',
+  SevenDays = '7 days',
+  TwentyEightDays = '28 days',
```

Comment:
suggestion: since these don't necessarily need to be human readable (AFAIK), what about keeping the right-side the same as the left (e.g `OneHour = 'oneHour'`) and using that as a `limitOrder.expiryOption.${key}` instead of `getExpiryOptionTranslation` method?

Reply:
Turbo tidy suggestion.

Reply:
yeah this is cool!